src/clandinin_lab_to_nwb/brezovec/brezovecimaginginterface.py

```diff
@@ -7,15 +7,288 @@
 from neuroconv.utils.dict import DeepDict
 
 
-class BrezovecMultiPlaneImagingInterface(BaseImagingExtractorInterface):
+class BrezovecFunctionalGreenImagingInterface(BaseImagingExtractorInterface):
     """
-    Data Interface for writing the Clandinin lab imaging data to NWB file using BrezovecMultiPlaneImagingExtractor.
+    Data Interface for writing the functional imaging data (GCaMP6f) for Clandinin lab to NWB file using BrezovecMultiPlaneImagingExtractor.
     """
 
     Extractor = BrezovecMultiPlaneImagingExtractor
 
     def __init__(self, folder_path: FolderPathType, stream_name: str, verbose: bool = True):
+        """
+        Initialize reading of NIfTI files.
+
+        Parameters
+        ----------
+        folder_path : FolderPathType
+            The path to the folder that contains the NIfTI image files (.nii) and configuration files (.xml) from Bruker system.
+        stream_name: str, optional
+            The name of the recording channel.
+        verbose : bool, default: True
+        """
+        super().__init__(
+            folder_path=folder_path,
+            stream_name=stream_name,
+            verbose=verbose,
+        )
+
+    @classmethod
+    def get_streams(cls, folder_path) -> dict:
+        streams = BrezovecMultiPlaneImagingExtractor.get_streams(folder_path=folder_path)
+        return streams
+
+    def get_metadata(self) -> DeepDict:
+        metadata = super().get_metadata()
+
+        xml_metadata = self.imaging_extractor.xml_metadata
+        session_start_time = parse(xml_metadata["date"])
+        metadata["NWBFile"].update(session_start_time=session_start_time)
+
+        description = f"Version {xml_metadata['version']}"
+        device_name = "BrukerFluorescenceMicroscope"  # TODO doucle check in the paper
+        metadata["Ophys"]["Device"][0].update(
+            name=device_name,
+            description=description,
+        )
+
+        imaging_plane_metadata = metadata["Ophys"]["ImagingPlane"][0]
+        imaging_plane_metadata.update(
+            # optical_channel =
```

Comment:
How can I hard-code the information on the emission lambda and description?

Reply:
Are them arguments of a specific object of `pynwb.ophys`? 

https://pynwb.readthedocs.io/en/stable/pynwb.ophys.html

If so, they should be in the metadata of the objects that corresponds to them in this function.

If they are not, then maybe we can add them as description of the most fitting object.

Emission lambda seems to be an argument of https://pynwb.readthedocs.io/en/stable/pynwb.ophys.html#pynwb.ophys.OpticalChannel
For example.